### PR TITLE
August casts

### PR DESCRIPTION
**Pull request by benfoxall**
The episode descriptions are **super boring**, very much missing @lpmi-13's wordsmithness.

@lpmi-13, we recorded a couple of longer episodes (6m & 20m). Maybe that's something we could mix in with the microterview episodes in the future?

### CODE REVIEW

File: feed.xml [hunk 1]
```diff
@@ -16,6 +16,34 @@
         <itunes:image href="https://remotehack.space/live/assets/big-hack.jpg" />
         <itunes:category text="Technology"/>
 
+
+        <item>
+            <title>Episode 19 - Remote Hack Macroterview</title>
```
Comment (lpmi-13) [inline feed.xml]: let's update this to have a very obvious "HackChat" somewhere in the title (maybe in brackets at the end?)

`Episode 19 - Ben and Ryan (HackChat)`
?

File: feed.xml [hunk 2]
```diff
@@ -16,6 +16,34 @@
...
+            <pubDate>Sat, 29 August 2020 15:00:00 +0000</pubDate>
+            <itunes:author>Remote Hack</itunes:author>
+            <itunes:duration>00:17:44</itunes:duration>
+            <itunes:explicit>no</itunes:explicit>
+            <guid>https://remotehack.space/live/feed.xml?episode=19</guid>
+        </item>
+
+
+        <item>
+            <title>Episode 18 - August Remote Hack</title>
```
Comment (lpmi-13) [inline feed.xml]: I guess something that's only 5 minute doesn't really feel long enough to be a "HackChat"...probably just keep this as is and maybe spice it up a tad...

`Episode 18 - Ben and Ryan steal the Mic and Break ALL THE RULES!`

File: feed.xml (hunk 1 above)
Comment (benfoxall) [reply feed.xml]: great :+1:

File: feed.xml (hunk 2 above)
Comment (benfoxall) [reply feed.xml]: 💥 🎤 ✋

**Pull request merged**